File: melosys-eessi-app/src/test/java/no/nav/melosys/eessi/models/DatoUtilsTest.java
package no.nav.melosys.eessi.models;

import java.time.LocalDate;

import org.junit.jupiter.api.Test;

import static no.nav.melosys.eessi.models.DatoUtils.tilLocalDate;
import static org.assertj.core.api.Assertions.assertThat;

class DatoUtilsTest {

    @Test
    void tilLocalDate_stringMedTime_parserTilLocaldate() {
        final String input = "2020-01-01:+02:00";
        final LocalDate forventet = LocalDate.of(2020, 1, 1);
        assertThat(tilLocalDate(input)).isEqualTo(forventet);

    }
}

File: melosys-eessi-app/src/test/java/no/nav/melosys/eessi/models/DatoUtilsTest.kt
package no.nav.melosys.eessi.models

import io.kotest.matchers.shouldBe
import no.nav.melosys.eessi.models.DatoUtils.tilLocalDate
import org.junit.jupiter.api.Test
import java.time.LocalDate

class DatoUtilsTest {

    @Test
    fun tilLocalDate_stringMedTime_parserTilLocaldate() {
        val input = "2020-01-01:+02:00"
        val forventet = LocalDate.of(2020, 1, 1)

        tilLocalDate(input) shouldBe forventet
    }
}
